docs/il-data-normalization.rst

```diff
@@ -0,0 +1,39 @@
+The traffic stop data received from the state of Illinois was post-processed
+and interpreted as described below.
+
+"agencyname" column
+-------------------
+
+The same agency may be written in multiple variations such as
+"ABINGDON POLICE" and "ABINGDON POLICE DEPARTMENT".  The first variation
+encountered in the dataset is the one used for display, after converting to
+mixed case.
+
+"Race" column
+-------------
+
+The input data uses the code "O" for "other".  The web interface refers to
+the "O" race as "unknown".
```

Comment:
This is how MD was handled, though it wasn't as striking since there were more non-other choices available.  I think it is safe to say that we don't know the race/ethnicity, even if the officer did.

Is it okay for the time being to say that we'll go through this explicitly with Ian?